starter/script.js

```diff
@@ -1 +1,143 @@
-// put code here!
+// Any path the user goes down must ask them at least three questions.
+
+// There must be a minimum of seven total destinations the user could arrive at based on their responses.
+
+// For at least one of the questions asked, there must be more than two possible user responses.
+
+// Your code must contain at least one loop.
+
+// Your code must make use of both string and number user inputs.
+
+var username = prompt("Welcome to DEAD OR ALIVE(DOA), you are a captured prisoner now. Tell us your name.");
+
+var greeting = alert("So you're " + username + "? Complete these 3 tasks and you'll walk out of here alive. Otherwise, imminent death awaits you.");
```

Comment:
when you do "var username = prompt('text');

You assign the user input from the prompt into the variable username.
However when it is an Alert('message")
There is no need to do "var greeting = "
just do "alert('Hey!')